README.md

```diff
@@ -10,9 +10,11 @@ For [evals orb](https://circleci.com/developer/orbs/orb/circleci/evals) usage ex
 
 ### Setup
```

Comment:
The set up does not mention that the user needs to set up context env variables for LangSmith and BrainTrust. We should add this!